Makefile

```diff
@@ -69,6 +69,10 @@ go.sum: go.mod
 		@echo "--> Ensure dependencies have not been modified"
 		GO111MODULE=on go mod verify
 
+# Print the value of a variable
+print-%:
+	@echo $*=$($*)
+
 # Uncomment when you have some tests
```

Comment:
we do have tests now :p

Reply:
yeah I saw that too, I'll just remove it